pages/common/pio-home.md

```diff
@@ -0,0 +1,24 @@
+# pio home
+
+> Launch the PlatformIO Home Web-server.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_home.html>.
+
+- Set specific HTTP port for the server (defaults to 8008):
+
+`pio home --port {{port}}`
+
+- Set specific HTTP host for the server (defaults to 127.0.0.1):
+
+`pio home --host {{host}}`
```

Comment:
What exactly does this example do?

Reply:
The usage example from the website is, that you can start your server using `0.0.0.0` as host for inbound connection. But I don't know how else this could be useful.

Reply:
Oh, that's call the bind address or listening address, in that you're binding to a specific interface. I'd suggest that the wording on the website may be incorrect here?

Reply:
What would you suggest?

Reply:
Hrm, what about this?
```suggestion
- Bind to a specific IP address (defaults to 127.0.0.1):

`pio home --host {{ip_address}}`
```

Reply:
Sounds good